src/cmap/connection.ts

```diff
@@ -170,7 +170,9 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
   address: string;
   socketTimeoutMS: number;
   monitorCommands: boolean;
+  /** True when after of onClose, onError, or onTimeout was called */
   closed: boolean;
+  /** True only after destroy has been called */
```

Comment:
these variables are states.  the documentation should focus not on what methods explicitly create that state (because the class internals change) but rather what the states are.

If we decide to update this documentation, I suggest something more along the lines of:
```typescript
/** Indicates that the connection (including the underlying socket) has been closed. */
closed: boolean
/** Indicates that the connection was explicitly destroyed. In the destroyed state, the connection is also closed. */
destroyed: boolean
```